R/plotLoadings.R

```diff
@@ -0,0 +1,385 @@
+#' Plot feature loadings after performing a reducedDim 
+#'
+#' Inspired by the \code{\link[diffTop:plotASVcircular]{plotASVcircular}} method using phyloseq 
+#' and has been converted to use TreeSummarizedExperiment objects.
```

Comment:
I would here include one sentence description of what this function does.

And then the current lines 3-4 could go to @details section.